p4a-recipes/zstandard/__init__.py

```diff
@@ -13,6 +15,14 @@ class ZStandardRecipe(CompiledComponentsPythonRecipe):
     )
     depends = ["cffi", "setuptools"]
 
+    # Local patches need to be an absolute paths or they won't be found
+    # by patch.
+    #
+    # https://github.com/kivy/python-for-android/issues/2623
```

Comment:
OK, according to the p4a PR it's closed and completed by another commit. So we should keep an eye on updating p4a and remove this patch.

Reply:
Yeah, I'm going to do a p4a rebase next and I'll drop this when I get there.

Reply:
Oh, I forgot that I already backported this to our p4a branch. Oh well, I'll just clean up post rebase.